glomap/math/gravity.h

```diff
@@ -14,4 +14,7 @@ double RotUpToAngle(const Eigen::Matrix3d& R_up);
 // Get the upright rotation matrix from a rotation angle
 Eigen::Matrix3d AngleToRotUp(double angle);
 
+// Estimate the average gravity direction from a set of gravity directions
+Eigen::Vector3d AverageGravity(std::vector<Eigen::Vector3d>& gravities);
```

Comment:
Does it make sense to write a few unit tests for this function?

Reply:
For average gravity? Or for rotation averaging? For average gravity, I think it is simply some SVD. For the rotation averaging, then I would need to design it a bit :)